UOP1_Project/Assets/Settings/Input/GameInput.cs

```diff
@@ -1,5 +1,5 @@
 // GENERATED AUTOMATICALLY FROM 'Assets/Settings/Input/GameInput.inputactions'
-
+// <auto-generated/>
```

Comment:
I believe this tag needs to be added manually right? If that's the case I think it may be a better idea to add generated code files to the `.editorconfig` file:

```
[Assets/Settings/Input/GameInput.cs]
generated_code = true
```

https://docs.microsoft.com/en-us/visualstudio/code-quality/use-roslyn-analyzers#configure-generated-code